fac29/NisaInvest-TFB-FE
# added shadow + fixed sticky nav bar

## Description
I've added a shadow to the nav bar and also fixed how the sticky position works, so you don't have to add a padding-top anywhere.

## Related Issue

## How Can This Be Tested?
Run the updated app, navigate to 'Book a demo' and you'll see the nav bar no longer overlaps the content.

## Checklist:
